### PR TITLE
Improve miniforge installation instructions

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: content/contributors/setup/install.md [hunk 1]
```diff
@@ -60,45 +60,50 @@ In [2]: np.linspace(0, 10, 5)
...
 A virtual environment is a workspace into which you can install Python
 libraries, separate from what is being used by your operating system.
 
 Create a new virtual environment in a directory called
-`py3`:
+`science` (or call it whatever you like!):
 
 ```
-mamba create -p py39
+mamba create -p science
```
Comment (jarrodmillman) [inline content/contributors/setup/install.md:92]:
```suggestion
conda create -p science
```

File: content/contributors/setup/install.md [hunk 2]
```diff
@@ -60,45 +60,50 @@ In [2]: np.linspace(0, 10, 5)
...
 ```
 
 Mamba uses `conda` to switch between virtual environments. Start using
 the new environment as follows:
```
Comment (jarrodmillman) [inline content/contributors/setup/install.md:96]:
```suggestion
Start using the new environment as follows:
```

File: content/contributors/setup/install.md [hunk 3]
```diff
@@ -60,45 +60,50 @@ In [2]: np.linspace(0, 10, 5)
...
 
 ```
-conda activate ./mamba39
+conda activate ./science
```
Comment (jarrodmillman) [inline content/contributors/setup/install.md:99]:
```suggestion
conda activate science
```

Is that needed? Does it work on Windows?

**Comment by jarrodmillman** [comment]
Double-check [07b69a7](https://github.com/scientific-python/learn.scientific-python.org/pull/47/commits/07b69a79d2bd0c4fea97ee355501ff548e0da0ac). I thought I was creating a PR.

**Comment by stefanv** [comment]
OK, fixed it up; I usually prefer directory names to create environments, but for most users using names will be better.

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**